neqo-bin/Cargo.toml

```
@@ -39,7 +38,7 @@ neqo-transport = { path = "./../neqo-transport" }
 neqo-udp = { path = "./../neqo-udp" }
 qlog = { workspace = true }
 quinn-udp = { workspace = true }
-regex = { workspace = true }
+regex = { workspace = true, features = ["unicode-perl", "perf"] }
```

```suggestion
regex = { workspace = true, features = ["unicode-perl"] }
```

Same here, preference for doing this in a separate pull request.